README.md

```diff
@@ -1,2 +1,127 @@
-# headers-generator
+# Header generator
 NodeJs package for generating browser-like headers.
+
+<!-- toc -->
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Examples](#examples)
+- [API Reference](#api-reference)
+
+<!-- tocstop -->
+
+## Installation
+Add this repository as an npm dependency and run the npm install command. No further setup is needed after that.
```

Comment:
Maybe we can add a command like this? 
```
npm install header-generator
```

Reply:
We would have to publish this as an npm package to do that, right?